### PR TITLE
Add Methods to user and post and likes

### PR DESCRIPTION
**Pull request by AACHOURMOHAMED**
# In this pull request
> Add a method that updates the comments counter for a post.
> Add A method that updates the likes counter for a post.
> Add A method that updates the posts counter for a user.
> Add A method that returns the 5 most recent comments for a given post.
> Add a method that returns the 3 most recent posts for a given user.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
```
Comment (danmainah) [inline app/models/user.rb]: - [ ] @AACHOURMOHAMED great work at creating your association, however, you forgot to add an association between the user and likes is missing in this model kindly add it.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @AACHOURMOHAMED, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/models/user.rb (hunk 1 above)
Comment (AACHOURMOHAMED) [reply app/models/user.rb]: @danmainah  Thank you 
I will

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @AACHOURMOHAMED , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions by the previous reviewer.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by AACHOURMOHAMED** [comment]
# Thank you for approving the project 👍 💯

**Pull request merged**